docs/source/geo_def.rst

```diff
@@ -88,10 +89,83 @@ based on area extent and a proj4-string or a list of proj4 arguments.
  Number of rows: 425
  Area extent: (-5326849.0625, -5326849.0625, 5326849.0625, 5326849.0625)
 
+The function **from_params** attempts to find shape and area_extent with the given data below:
```

Comment:
I think we should start using sphinx's links to python objects when possible. See https://raw.githubusercontent.com/pytroll/satpy/master/doc/source/dev_guide/xarray_migration.rst for some examples. I can't find the actual documentation for this but pycharm does a pretty good job of autocompleting if I remember correctly. If you add this syntax for the first use or two of a python object (function, method, class, etc) it should give the user a nice obvious way to get to the API docs.